vllm/worker/vineyard_llm_cache.py

```diff
@@ -301,8 +346,10 @@ def prefetch_seq_kv_caches(
 
         if query_token_size <= 0:
             return seq_id, 0
-
-        start_time = time.perf_counter()
+        if self.metrics_enabled:
```

Comment:
Inexpensive metrics such as `total_tokens` and `total_blocks` should always be collected. The `metrics_enabled` flag should only protect expensive metrics that have potential performance impact, such as those involving calling `cuda.synchronize()`.